### PR TITLE
V3 Migration build Prep

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15560

*Description of changes:*
* Major reformatting of all Class Based Components to Composition API. 
* Minor test udates. 

*NOTE:*
- Whats left is a few Shared components (Staff Payment, DatePicker) etc, that will need to be localized in order to remove the package dependencies completely, in conjunction with Mixins.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/parties/party/SecuredParties.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-container fluid no-gutters id="secured-parties-component" class="pa-0">
-    <change-secured-party-dialog
+    <ChangeSecuredPartyDialog
```
Comment (dimak1) [inline ppr-ui/src/components/parties/party/SecuredParties.vue:3]: Nice to (finally) see full names without dashes

**Review by dimak1 — APPROVED** [review]
Great refactor!

File: ppr-ui/src/components/parties/party/SecuredParties.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/parties/party/SecuredParties.vue:3]: Yes! Will try and catch these as we go :) 
I haven't looked at the templates too much yet but upgrading Vuetify we will definitely be in there 💥

**Review by doug-lovett — APPROVED** [review]
Scanned the files with large number of differences. Looks great, a lot of work.

File: ppr-ui/src/components/parties/party/SecuredParties.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/parties/party/SecuredParties.vue:3]: Awesome, thanks!

**Pull request merged**